#! /bin/sh /usr/share/dpatch/dpatch-run
## 90_cdda-1.3.38.dpatch by Thomas Günther <tom@toms-cafe.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes for VDR >= 1.3.38.

@DPATCH@
--- cdda-0.1.0/audio_player.c
+++ cdda-0.1.0/audio_player.c
@@ -368,7 +368,11 @@
 }
 
 cAudioControl::~cAudioControl() {
+#if VDRVERSNUM >= 10338
+  cStatus::MsgReplaying(this, NULL, NULL, false);
+#else
   cStatus::MsgReplaying(this, NULL);
+#endif
   free(mp_replayDescription);
 
   delete player;
@@ -469,7 +473,11 @@
     mp_replayDescription = p_str;
     p_str = NULL;
 
+#if VDRVERSNUM >= 10338
+    cStatus::MsgReplaying(this, mp_replayDescription, NULL, true);
+#else
     cStatus::MsgReplaying(this, mp_replayDescription);
+#endif
   } else
     free(p_str);
 
